Add SysTimeModel

luizzmizz opened the pull request:
I added a new Instance::SysTimeModel package in order to retrieve v$sys_time_model data, which is really useful for drawing database activity. As you can see, I'm pretty newbie on perl and I needed three commits :(, even though this is a clone of your sysstats package

lausser commented:
This looks like a copy-paste-bug:
  ['server::instance::systimemodel::listsystimemodel',
      'list-systimemodel', undef,
      'convenience function which lists all statistics from v$syswaitclass' ],

Commit:
Apply PR #40 (SysTimeModel, SysWaitClass) + ASM mirror-aware usage fix

- Add SysTimeModel and SysWaitClass instance modes (from PR #40)
- Fix copy-paste bug in list-systimemodel mode description (v$syswaitclass -> v$sys_time_model)
- Fix ASM diskgroup usage% to account for NORMAL/HIGH redundancy mirror factors
  using required_mirror_free_mb and effective_total instead of raw total_mb
- Add BUILD.md with build instructions including cross-platform perl path note
